Python/clinic/sysmodule.c.h

```diff
@@ -509,15 +509,6 @@ sys_get_coroutine_origin_tracking_depth(PyObject *module, PyObject *Py_UNUSED(ig
     return return_value;
 }
 
-PyDoc_STRVAR(sys_set_coroutine_wrapper__doc__,
```

Comment:
I believe this needs to rerun clinic as seen in CI.

```
➜  cpython git:(pr_13577) ✗ make clinic
python3 ./Tools/clinic/clinic.py --make --srcdir .
Error in file "./Python/sysmodule.c" on line 3305:
Checksum mismatch!
Expected: 3c32bc91ec659509
Computed: 7bd9d8b6cb595ff4
Suggested fix: remove all generated code including the end marker,
or use the '-f' option.
make: *** [clinic] Error 255
```

* Run clinic with -f option and commit the changed file

```
➜  cpython git:(pr_13577) python3 ./Tools/clinic/clinic.py -f --make --srcdir .
➜  cpython git:(pr_13577) ✗ gst
On branch pr_13577
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   Python/clinic/sysmodule.c.h

no changes added to commit (use "git add" and/or "git commit -a")
```

* Patch for reference on running the above command

```diff
diff --git a/Python/clinic/sysmodule.c.h b/Python/clinic/sysmodule.c.h
index 5f4b451ca9..61bdcfb0ea 100644
--- a/Python/clinic/sysmodule.c.h
+++ b/Python/clinic/sysmodule.c.h
@@ -1099,4 +1099,4 @@ sys_getandroidapilevel(PyObject *module, PyObject *Py_UNUSED(ignored))
 #ifndef SYS_GETANDROIDAPILEVEL_METHODDEF
     #define SYS_GETANDROIDAPILEVEL_METHODDEF
 #endif /* !defined(SYS_GETANDROIDAPILEVEL_METHODDEF) */
-/*[clinic end generated code: output=3c32bc91ec659509 input=a9049054013a1b77]*/
+/*[clinic end generated code: output=7bd9d8b6cb595ff4 input=a9049054013a1b77]*/
```

Reply:
Done.